### PR TITLE
README: note a couple more kconfigs

### PR DESCRIPTION
**Pull request by bluca**

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -128,6 +128,10 @@ REQUIREMENTS:
 
         Required for signed Verity images support:
           CONFIG_DM_VERITY_VERIFY_ROOTHASH_SIG
+        Required to verify signed Verity images using keys enrolled in MoK:
```
Comment (yuwata) [inline README]: What is MoK?

Comment (bluca) [reply README]: machine-owner-keyring as set up by shim

Comment (yuwata) [reply README]: Is it a famous abbreviation? I've never heard that. Could you mention that explicitly?

**Comment by DaanDeMeyer** [comment]
Can you add these to the mkosi build script kernel stuff as well (no need to test it explicitly, it's just nice to have them recorded there as well).

File: mkosi.build [hunk 2]
```diff
@@ -307,6 +307,10 @@ if [ -d mkosi.kernel/ ]; then
                 --enable MEMCG \
                 --enable MEMCG_SWAP \
                 --enable MEMCG_KMEM \
+                --enable IMA_ARCH_POLICY \
+                --enable DM_VERITY_VERIFY_ROOTHASH_SIG \
+                --enable DM_VERITY_VERIFY_ROOTHASH_SIG_SECONDARY_KEYRING \
+                --enable CONFIG_INTEGRITY_MACHINE_KEYRING \
```
Comment (daandemeyer) [inline mkosi.build]: This should just be `INTEGRITY_MACHINE_KEYRING`

Comment (bluca) [reply mkosi.build]: whops - as said, 100% untested :-)

File: README [hunk 3]
```diff
@@ -128,6 +128,11 @@ REQUIREMENTS:
 
         Required for signed Verity images support:
           CONFIG_DM_VERITY_VERIFY_ROOTHASH_SIG
+        Required to verify signed Verity images using keys enrolled in the MoK
+        (Machine-Owner Key) keyring:
+          CONFIG_DM_VERITY_VERIFY_ROOTHASH_SIG_SECONDARY_KEYRING
+          CONFIG_IMA_ARCH_POLICY
+          CONFIG_CONFIG_INTEGRITY_MACHINE_KEYRING
```
Comment (yuwata) [inline README]: CONFIG_CONFIG_ ??

**Review by DaanDeMeyer — APPROVED** [review]

File: README (hunk 3 above)
Comment (bluca) [reply README]: it configures a lot

**Pull request merged**